components/drata/sources/new-evidence-added/new-evidence-added.mjs

```diff
@@ -7,7 +7,7 @@ const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/
 export default {
   key: "drata-new-evidence-added",
   name: "New Evidence for Control Added",
-  description: `Emit new event for every new evidence for a control. [See docs here](${docsLink}).`,
+  description: `Emit a new event for every new evidence for a control. [See docs here](${docsLink}).`,
```

Comment:
```suggestion
  description: `Emit a new event for every new evidence for a control. [See the documentation](${docsLink}).`,
```

Reply:
Changed "docs here" to "the documentation" to follow Google's Developer docs style guide.